Source/LES.cpp

```diff
@@ -326,7 +326,7 @@ PeleC::getSmagorinskyLESTerm(
         reflux_factor, LESTerm, dt, mfi, typ,
         {AMREX_D_DECL(&flux_ec[0], &flux_ec[1], &flux_ec[2])});
     } // End of MFIter scope
-  }   // End of OMP scope
+  } // End of OMP scope
```

Comment:
this is likely due to a slight version difference between your clang-format and the version used in the testing. But you didn't actually change anything in this file, so just revert this back to what it was.

Reply:
Whoops sorry about that. I reverted this back to the previous commit.